swap/src/state.rs

```diff
@@ -106,9 +107,10 @@ impl Display for Bob {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
             Bob::Negotiated { .. } => f.write_str("Handshake complete"),
-            Bob::BtcLocked { .. } | Bob::XmrLocked { .. } | Bob::BtcCancelled(_) => {
-                f.write_str("Bitcoin refundable")
-            }
+            Bob::BtcLocked { .. }
+            | Bob::XmrLocked { .. }
+            | Bob::T1Expired(_)
+            | Bob::BtcCancelled(_) => f.write_str("Bitcoin refundable"),
```

Comment:
Not sure why we started to combine these states in the first place.
Please have a display implementation per state. The discrepancy can only lead to confusion.

also fix `BtcRedeemed` while you are there :)

Reply:
Thanks! I should have actioned that one right away